pkg/operator/configobserver/config_observer_controller.go

```diff
@@ -154,3 +155,26 @@ func (l *listerInformer) AddEventHandler(cache.ResourceEventHandler) {
 func (l *listerInformer) HasSynced() bool {
 	return l.cacheSynced()
 }
+
+// WithPrefix adds a prefix to the path the input observer would otherwise
+// observe into
+func WithPrefix(observer ObserveConfigFunc, prefix ...string) ObserveConfigFunc {
+	if len(prefix) == 0 {
+		return observer
+	}
+
+	return func(listers Listers, recorder events.Recorder, existingConfig map[string]interface{}) (map[string]interface{}, []error) {
+		orig, errs := observer(listers, recorder, existingConfig)
```

Comment:
you have to project the existingConfig too

Reply:
ah, I didn't realize that, thanks